### PR TITLE
docs(roadmap): Remove roadmap doc FRMW-13

### PR DESCRIPTION
**Pull request by parlarjb**
Changes for https://jira.rax.io/browse/FRMW-13

### CODE REVIEW

**Comment by CITguy** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -57,9 +50,16 @@ For versioning, we rougly follow the [Semantic Versioning system](http://semver.
 
 A breaking change is one that *requires* users of the changed component to modify their own code.
 
+### How are issues prioritized?
+
+As a small team with limited resources, Encore relies on prioritization to drive development. However, EncoreUI is open source and new components or functionality may be submitted in a pull request. Still, there are times when EncoreUI's team must review or tweak design and functionality of features before they are ready to be brought in. Please see our [Contribution Guide](./guides/CONTRIBUTING.md) if you are planning on submitting a pull request.
```
Comment (TheNando) [inline README.md]: The file is at the root, not in /guides. Though I think it should be i guides, and the name all lowercase.

Comment (parlarjb) [inline README.md]: Having it capitalized and in the root is a github convention: https://github.com/blog/1184-contributing-guidelines

But I'll fix this line, thanks for the catch

**Comment by megalord** [comment]
LGTM

**Pull request merged**